docs/application/dotnet/get-started/first-app.md

```diff
@@ -81,25 +81,25 @@ To create a new Tizen .NET project, you can use following tools:
                     <ol>
                         <li>Launch the Visual Studio Code tool.</li>
                         <li>Use <b>Ctrl + Shift + P</b> to open Command Palette and type <b>Tizen Create</b> and hit Enter key.</li>
-                        <img alt=" " src="media/vscode_create_1.png"/>
+                        <img alt="Create a project" src="media/vscode_create_1.png"/>
                         <p></p>
                         <li>Select project type: <b>DOTNET</b>.</li>
-                        <img alt=" " src="media/vscode_create_2.png"/>
+                        <img alt="Project type selection" src="media/vscode_create_2.png"/>
                         <p></p>
                         <li>Select your preferred device profile.</li>
-                        <img alt=" " src="media/vscode_create_3.png"/>
+                        <img alt="Device profile selection" src="media/vscode_create_3.png"/>
                         <p></p>
                         <li>Select profile version.</li>
-                        <img alt=" " src="media/vscode_create_4.png"/>
+                        <img alt="Device profile version selection" src="media/vscode_create_4.png"/>
                         <p></p>
                         <li>Choose your desired project template.</li>
-                        <img alt=" " src="media/vscode_create_5.png"/>
+                        <img alt="Select project template" src="media/vscode_create_5.png"/>
                         <p></p>
-                        <li>Choose the Tizen project name. In this case, <b>HelloWorld</b> is used.</li>
-                        <img alt=" " src="media/vscode_create_6.png"/>
+                        <li>Enter your Tizen project name. In this case, <b>HelloWorld</b> is being used.</li>
```

Comment:
Please refer https://developers.google.com/style/tense and https://developers.google.com/style/voice?hl=en

Reply:
@mijin85cho I have changed voice from passive to active. Can you check if my implementation is alright?

Reply:
I think present tense is recommended rather than present progressive.

https://developers.google.com/style/tense

Also, would you please update the remaining parts requested in https://github.com/Samsung/tizen-docs/pull/1599 and 1590?
There are still many things to be updated.